weblate/addons/models.py

```diff
@@ -68,22 +86,32 @@ class Meta:
         verbose_name = "add-on"
         verbose_name_plural = "add-ons"
 
-    def __str__(self) -> str:
-        return f"{self.addon.verbose}: {self.component}"
+    def __str__(self):
```

Comment:
Type hint gone missing:

```suggestion
    def __str__(self) -> str: 
```